### DISCUSSION TITLE
some thoughts and problems

### DISCUSSION THREAD

**Discussion by snektailmeat** [NuclearPhoenixx/Kobra_Neo#3] [Ideas]
would it be possible to enable input shaping (or even linear advance if it works now) ? i tried and build it myself but one package doesnt seem to install. oh and i think saving the preheat pla setting doesnt work right, it always stays at 190c

**Comment by NuclearPhoenixx** [comment]
Hi there, linear advance doesn't work at the moment. I've not tested input shaping and honestly I'd be surprised if it works on this ancient and modified version of Marlin. If you already tried to compile it and it didn't work, I think that's as far as you'll be able to go. They removed most vanilla Marlin code that wasn't relevant for them at the time and adding it back is a pain, because they did some modifications to the rest of the code base where they hard-coded things and so on. The code base really is a mess and you probably won't get any new features out of it if nobody comes up with a miracle and gets a vanilla Marlin build to work on the Neo. At this point it's more about fixing bugs and other smaller things, sorry.

No idea about the preheating as I never use that, you'll have to give me some more detail on that.

**Reply by snektailmeat** [reply]
hi, as for the compiling i havent even gotten that far i get an error in the pack installer"Cannot install Pack(HDSC:HC32F460:1.0.11), Pack not found"

as for the preheating, in the configuration page you can set the preset temperature but it doesnt seem to apply ist still the stock 190,60.

**Reply by NuclearPhoenixx** [reply]
Bruh sorry for talking straight but did you even read the README? It literally says it right there that linear advance doesn't work and it also links a tutorial on how to build the firmware.

Note sure about the preheat issue. If you change it and it still stays the same, there might be something hard-coded by Anycubic. If you can tell me exactly what you did to get this issue, how to replicate it, what it looks like and if it happens all the time (i.e. repeatable), you are welcome to open an issue and I'll have a look.
